renovate.json

```
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [":pinAllExceptPeerDependencies"],
+  "semanticCommits": "enabled",
+  "enabledManagers": ["npm", "docker"],
+  "branchPrefix": "deps/",
+  "timezone": "Europe/Berlin",
+  "prConcurrentLimit": 20,
+  "schedule": ["every day at 06:00"],
```

That's too often. Lets reduce once per week.
```suggestion
  "schedule": ["every weekend"],
```